packages/builder/src/userInterface/ComponentPanel.svelte

```diff
@@ -42,28 +42,29 @@
         <LayoutIcon />
       </button>
     </li>
-    <li>
-      <button
-        class:selected={current_view === 'code'}
-        on:click={() => codeEditor && codeEditor.show()}>
-        {#if component._code && component._code.trim().length > 0}
-          <div class="button-indicator">
-            <CircleIndicator />
-          </div>
-        {/if}
-        <TerminalIcon />
-      </button>
-    </li>
-    <li>
-      <button
-        class:selected={current_view === 'events'}
-        on:click={() => (current_view = 'events')}>
-        <EventsIcon />
-      </button>
-    </li>
+    {#if !component._component.startsWith('##')}
```

Comment:
just to clarify - this means that all built-ins wil appear in the components list, regardless of which library is selected ? I'm fine with this, just checking

Reply:
This particular one means that any `##` prepended built-ins won't get a code or events button in the sidebar.

[This one](https://github.com/Budibase/budibase/pull/109/files#diff-29c8e2f7ea7bcaa19f3ccb7267d43c84R69) means that all builtins will show in addition to the current components.